roles/lib_utils/action_plugins/generate_pv_pvcs_list.py

```diff
@@ -118,10 +118,16 @@ def build_pvc_dict(self, varname=None):
                     create_pvc = self._templar.template(create_pvc)
                     if kind != 'object' and create_pv and create_pvc:
                         volume, size, _, access_modes = self.build_common(varname=varname)
+                        storageclass = self.task_vars.get(str(varname) + '_storageclass')
+                        if storageclass:
+                            storageclass = self._templar.template(storageclass)
+                        elif not storageclass and kind != 'dynamic':
```

Comment:
There's a chance that 'false' string is passed in and resolves to boolean False.  Is it expected that a user might try to put a boolean value in this variable?  If not, we're good.  Otherwise, we should remove the 'elif' in favor of another if.